src/fontique/backend/android.rs

```diff
@@ -87,11 +94,59 @@ impl SystemFonts {
                                         //       smarter, or something dumb that meets expecta­
                                         //       tions on Android.
                                     }
-                                } else if let Some(_langs) = child
+                                } else if let Some(langs) = child
                                     .attribute("lang")
                                     .map(|s| s.split(',').collect::<Vec<&str>>())
                                 {
-                                    // TODO: implement language fallback "family" elements
+                                    let (_has_for, hasnt_for): (Vec<Node>, Vec<Node>) = child
+                                        .children()
+                                        .partition(|c| c.attribute("fallbackFor").is_some());
+                                    {
+                                        // general fallback families
+                                        let (ps_named, _ps_unnamed): (Vec<Node>, Vec<Node>) =
+                                            hasnt_for.iter().partition(|c| {
+                                                c.attribute("postScriptName").is_some()
+                                            });
+
+                                        if let Some(family) = ps_named.iter().find_map(|x| {
+                                            postscript_names
+                                                .get(x.attribute("postScriptName").unwrap())
+                                        }) {
+                                            for lang in langs {
+                                                if let Some(scr) = lang.strip_prefix("und-") {
+                                                    script_fallback.push((scr.into(), *family));
+                                                } else if let Ok(locale) =
+                                                    LanguageIdentifier::try_from_bytes(
+                                                        lang.as_bytes(),
+                                                    )
+                                                {
+                                                    if let Some(scr) = locale.script {
+                                                        script_fallback
+                                                            .push((scr.as_str().into(), *family));
+                                                        if "Hant" == scr.as_str() {
+                                                            // This works around ambiguous han char­
+                                                            // acters going unmapped with current
```

Comment:
I'm not sure I'd put a line break there.
This is also getting very nested, although I haven't reviewed enough surrounding code to see that there's an easy boundary to break things up

Reply:
It's the 99 column limit, my impression was that we were keeping that but...